data/json/items/gun/9mm.json

```diff
@@ -115,7 +115,7 @@
     "durability": 6,
     "magazine_well": 1,
     "magazines": [
-      [ "9mm", [ "glockmag", "glockbigmag" ] ],
+      [ "9mm", [ "glockmag", "glockbigmag", "glock17_17", "glock_drum_50rd", "glock_drum_100rd" ] ],
```

Comment:
I'm sure the same applies to the 22 round mag. It could be added here.

Reply:
Okay.